src/azure-cli/azure/cli/command_modules/storage/azcopy/util.py

```diff
@@ -164,7 +164,7 @@ def _get_default_install_location():
             return None
         install_location = os.path.join(home_dir, r'.azcopy\azcopy.exe')
     elif system in ('Linux', 'Darwin'):
-        install_location = '/usr/local/bin/azcopy'
+        install_location = '/home/bin/azcopy'
```

Comment:
This should work. Consider OSX's user folder is `/users/<name>`,  to get 100% worry free, consider use `~` like [here](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/azure/cli/core/_environment.py#L9).